### DISCUSSION TITLE
Can we use custom domain for Budibase apps if we deploy on Digital Ocean ?

### DISCUSSION THREAD

**Discussion by satriowicaksn** [Budibase/budibase#2737] [Help]
i have deployed budibase to digital ocean, and my access url is ipv4 167.172.86.73:10000/builder/portal/apps. can i change that ip to a domain name like example.com ?

**Comment by chrisshaw** [comment]
Yep! You can do so in multiple ways. This isn't anything specific to Budibase.

I would suggest taking a look at [Digital Ocean's documentation](https://docs.digitalocean.com/products/networking/dns/) on managing domains. The [quickstart](https://docs.digitalocean.com/products/networking/dns/quickstart/) will walk you through the basic steps.

How you do it depends on who owns your custom domain. This [third-party article](https://kaloraat.com/articles/add-domain-to-digital-ocean-droplet) (which I plucked relatively at random) seems to do okay walking through the steps with Namecheap. This [third-party video](https://www.youtube.com/watch?v=QduZyoy_eFA) promises to cover all the important steps (plus a few you don't need).
